tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I have also moved the config files to the root of the repository since src directories normally are only for the TS files.  The compiler and linting options should be global for the whole repo anyway and this doesn't hide them.